pkg/dwarf/line/line_parser.go

```diff
@@ -3,9 +3,12 @@ package line
 import (
 	"bytes"
 	"encoding/binary"
+	"io/ioutil"
 	"path/filepath"
 
 	"github.com/derekparker/delve/pkg/dwarf/util"
+	"github.com/derekparker/delve/pkg/logflags"
+	"github.com/sirupsen/logrus"
```

Comment:
IMHO a reusable package like this shouldn't depend on a specific logging library that's not part of the standard library. Especially given the talk of contributing it to x/debug/dwarf.

Same goes for depending on logflags.

Reply:
All good points mentioned above, maybe we can change the package to accept a logger function, which will not add any imports.

Reply:
I think that would be good.